src/orion/serving/responses.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.serving.responses` -- Build JSON-compatible responses for Orion objects
+===================================================================================
+
+.. module:: responses
+   :platform: Unix
+   :synopsis: Build JSON-compatible responses for Orion objects
+"""
+from orion.core.worker.trial import Trial
+
+
+def build_trial_response(trial: Trial) -> dict:
+    """
+    Build the response representing a trial according to the API specification.
+
+    Parameters
+    ----------
+    trial: Trial
+        The trial to return for the API.
+
+    Returns
+    -------
+    A JSON-serializable dict representing the given trial.
+
+    """
+    return {'id': trial.id,
```

Comment:
It may be worth adding `experiment` for cases where trials are fetched within the EVC tree, otherwise we loose its provenance. Do we plan on using this function for such calls? Otherwise I think the trial response covers everything.

Reply:
The goal of this function is to strictly transform Orion's data objects into the corresponding Web API specification for representing a trial. 

Including `experiment` in the description, might be redundant since a trial is either queried under the `bestTrial` key in the endpoint `experiments/:name` or as the root dictionary in the `trials/:experiment_name/:id` endpoint. In both cases, the caller specifies the experiment so he should already know which experiment is associated with the trial.